snap/snapcraft.yaml

```diff
@@ -5,8 +5,78 @@ summary: Snap for NVIDIA DCGM and DCGM exporter
 license: Apache-2.0
 contact: solutions-engineering@lists.canonical.com
 description: |
-  This snap includes NVIDIA DCGM and DCGM exporter to provide GPU monitoring via
-  Prometheus metrics. This snap does not include the dcgmproftester.
+  This snap includes NVIDIA DCGM and DCGM-exporter to monitor GPU via Prometheus metrics
+  A use can setup a Grafana dashboard to visualize the metrics.
+
+  The snap includes the following components:
+  - DCGM: Data Center GPU Manager
+  - DCGM-exporter: A Prometheus exporter for DCGM metrics
+  - NV-Hostengine: A service that communicates with the NVIDIA GPU driver to get the metrics
+
```

Comment:
I suggest immediately mentioning where to find documentation about the included components. For example:
```
Please see the links at the bottom of the page for more details about the included components.
```